proto/config/v1/config.proto

```diff
@@ -1,11 +1,11 @@
 syntax = "proto3";
 
-package org.hypertrace.agent.config;
+package hypertrace.agentconfig.config.v1;
```

Comment:
The reason for this change is:
- Consistency across languages (see https://github.com/hypertrace/agent-config/pull/62/files#diff-2d841886dfa8c1ef2abf7d2df5b3abcd833443a1b322eb00a4a700aba56c0915R7-R8)
- It will be easy to introduce another section called API as per @davexroth's comment https://github.com/hypertrace/agent-config/pull/62#discussion_r613631714